src/run.js

```diff
@@ -50,27 +50,40 @@ const getSha = (eventName, payload) => {
   return null
 }
 
+const getBranchFromGithubRef = (ref) => {
+  const tokens = ref.split('/')
+  if (ref.startsWith('refs/') && tokens.length > 2) {
```

Comment:
should not be tokens.length >= 2 to allow the case with only 3 tokens ref/*/*?

Reply:
If you look `ref` field in the context [document](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context). There can be two format i.e. `refs/heads/<branch_name>` and `refs/pull/<pr_number>/merge`. In both case, we want third value in the expression always.

Reply:
Got it